ci/unit.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.13.8
```

Comment:
1.15?

Reply:
I left as I wasn't sure if all the libraries were tested to 1.15 yet.  But I'll change to 1.15 thanks :)